homeassistant/const.py

```diff
@@ -26,6 +26,7 @@
 CONF_ADDRESS = 'address'
 CONF_AFTER = 'after'
 CONF_ALIAS = 'alias'
+CONF_ALTITUDE = 'altitude'
```

Comment:
Please move this constant to the platform.

Reply:
Made the changes